index.html

```diff
@@ -265,7 +265,7 @@ <h1>
         <li data-lang="en" id="attomos.dev" data-owner="attomos">
           <a href="https://attomos.dev">attomos.dev</a>
         </li>
-        <li data-lang="en" id="edocstudio.info" data-owner="edocstudio">
+        <li data-lang="en" id="edocstudio.info" data-owner="edocstudio" data-feed="https://edocstudio.ngrok.io/feeds/release.rss">
```

Comment:
@edocstudio Can you put the feed on your domain name `edocstudio.info`? That URL can then redirect to `ngrok` if that’s what you prefer.

Reply:
Thank you for your recommend. I will try it :)

Reply:
Hi, Still got the same error. Please tell me if you have another way to solve it.

Reply:
@edocstudio Sorry for slow response. Don’t worry about E2E build error, it doesn't have anything to do with your Pull Request; I haven’t fixed the problem yet.

Your feed seems to not be valid. Try going to https://validator.w3.org/feed/ and fill in the feed URL.

<img width="698" alt="image" src="https://user-images.githubusercontent.com/193136/164089747-3fbbe656-7876-490c-b640-3360f9846b85.png">

Make sure that when you put in your feed URL, you get a “Congratulations!” message, which looks like this:

<img width="673" alt="image" src="https://user-images.githubusercontent.com/193136/164089896-f70a2e92-5d05-460a-a794-9d99e864d3ea.png">

Reply:
Thanks you for solution, I try to pass url from ngrok directly "**https://edocstudio.ngrok.io/feeds/release.rss**". It's work. So, index.html I should be use url from ngrok or url that redirect (**https://edocstudio.io/#/feeds/**) in data-feed.

![Screenshot from 2022-04-20 07-56-16](https://user-images.githubusercontent.com/101785602/164125742-fa918e22-bcce-474b-b809-4d946afe3135.png)

Reply:
@edocstudio Since you are using ngrok, you can [set up a Custom Domain for ngrok](https://ngrok.com/docs/guides/how-to-set-up-a-custom-domain).

Reply:
Thank you so much.